src/Files.FullTrust/MessageHandlers/DriveHandler.cs

```diff
@@ -16,16 +16,18 @@ public class DriveHandler : Disposable, IMessageHandler
 
         public void Initialize(PipeStream connection) {}
 
-        public async Task ParseArgumentsAsync(PipeStream connection, Dictionary<string, JsonElement> message, string arguments)
+        public Task ParseArgumentsAsync(PipeStream connection, Dictionary<string, object> message, string arguments)
```

Comment:
I think this line may have some merging issue. The message should be `Dictionary<string, JsonElement>`.

Reply:
DriveHandler is supposed to be removed in the upstream so I merged the changes in the latest commit.